monitoring-server/cmake/Project.cmake

```diff
@@ -11,6 +11,7 @@ target_sources(Core
         include/core/Device.h
         include/core/Event.h
         include/core/EventScheduler.h
+        include/core/Log.h
```

Comment:
Please refer to the first comment in "monit/core: Create Device class definition" commit for similar question.

Reply:
Same as above